### DISCUSSION TITLE
Alternative for Super Mode in Pyarmor 8?

### DISCUSSION THREAD

**Discussion by Thoufak** [dashingsoft/pyarmor#1697] [General]
I've looked through the docs and changelogs, but couldn't find any traces of the Super Mode (--advanced 2) in Pyarmor 8. Is there an alternative and what are the differences with the original Super Mode? Thanks.

**Comment by jondy** [comment]
There is no super mode in Pyarmor 8. Pyarmor 8 is rewitten, the runtime core is totally different from Pyarmor 7.
In Pyarmor 8, the recommanded mode is RFT mode and BCC mode.

**Comment by Thoufak** [comment]
Thanks
